cmd/golangBuild.go

```diff
@@ -631,3 +584,20 @@ func isMainPackage(utils golangBuildUtils, pkg string) (bool, error) {
 
 	return true, nil
 }
+
+func gitConfigurationForPrivateModules(privateMod string, token string, utils golangBuildUtils) error {
+	privateMod = strings.ReplaceAll(privateMod, "/*", "")
+	privateMod = strings.ReplaceAll(privateMod, "*.", "")
```

Comment:
Have you tested this case?
The `git config` [docs](https://git-scm.com/docs/git-config#Documentation/git-config.txt-urlltbasegtinsteadOf) say that *Any URL that **starts** with this value will be rewritten to start, instead, with \<base>.*

Reply:
let me test this case